thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -7,17 +7,77 @@ import { toCurrentUrn, toBaselineUrn, filterPrefix, dateFormatFull } from 'third
 import { checkStatus } from 'thirdeye-frontend/utils/utils';
 import _ from 'lodash';
 
-const queryParamsConfig = {
-  refreshModel: false,
-  replace: false
+const ROOTCAUSE_SETUP_MODE_CONTEXT = "context";
+const ROOTCAUSE_SETUP_MODE_SELECTED = "selected";
+const ROOTCAUSE_SETUP_MODE_NONE = "none";
+
+/**
+ * converts RCA backend granularity strings into units understood by moment.js
+ */
+const toMetricGranularity = (attrGranularity) => {
+  let [count, unit] = attrGranularity.split('_');
+
+  switch (unit) {
```

Comment:
I personally prefer the shorter way:

```
const UNIT_MAPPING = {
  MINUTES: 'minute',
  HOURS: 'hour',
  DAYS: 'day'
}
then UNIT_MAPPING[unit];
```

This also allows moving this constant to the `constants.js` file (if needed)

Reply:
fixed